index.html

```diff
@@ -101,6 +102,74 @@ <h2>
         display modes. Otherwise, the [=manifest/display=] member is sufficient
         for most use cases.
       </p>
+
+      <section>
+        <h3>Concepts</h3>
+        <ul>
+          <li>
+            <dfn data-lt="controls of the window">Window controls</dfn>:
+            interface elements that the operating system uses consistently across
+            applications to enable the user to perform certain actions to control
+            the application. Common actions in the [=window controls=] include
+            minimize, maximize/restore, and close buttons.
+          </li>
+          <li>
+            <dfn>Additional Window Controls</dfn>: feature to replace
+            [=window controls=] by calling the minimize/maximize/close etc APIs
+            directly without the need to use browser's native window controls.
+          </li>
+          <li>
+            <dfn data-lt="IWA">Isolated Web App</dfn>: feature for installed
+            and signed web applications which are packaged as Web Bundles and
+            can be used offline instead of being fetched over HTTPS.
+            More: https://github.com/WICG/isolated-web-apps
+          </li>
+          <li>
+            <dfn data-lt="draggable region">Draggable regions</dfn>. To define
+            which regions or elements in the title bar are draggable, this
+            specification defines the CSS <code><dfn>app-region</dfn></code>
+            property.
+          </li>
+        </ul>
+      </section>
+
+      <section>
+        <h3>
+          Display mode extensions
+        </h3>
+        <p>
+          Additionally to the normal
+          <a data-cite="appmanifest#dfn-display-modes-list">display modes</a>,
+          [=manifest/display_override=] also supports certain extensions to it.
+        </p>
+        
+        <dt>
+          <dfn data-export="" data-dfn-for="display mode">
+            borderless
+          </dfn>
+        </dt>
+        <dd>
+          The web application opens without any host-native title bar or
+          [=window controls=] visible and with the web contents extended to the
+          whole title bar area. The app will also be able to specify draggable
+          regions in the web contents to create a customized title bar.
+          Borderless mode defaults to `standalone` if it's used in a
+          non-Isolated Web App (IWA) or without Window Management permission.
```

Comment:
This shouldn't be specified like this - if the user agent doesn't support 'borderless', then it goes to the next display mode in the `display_override` list.

(e.g. If that is `standalone`, then great. Otherwise it defaults to using `display`, and if that is empty, it defaults to `browser`)